generators/server/templates/src/main/resources/config/_bootstrap.yml

```diff
@@ -0,0 +1,8 @@
+spring:
```

Comment:
you need to add this to the `profile_dev|prod.gradle` file in `processResources` section else value will not be replaced. not sure if anything is required for maven